### ISSUE TITLE
Add new placementStr to placement option: fixedbottom

### ISSUE THREAD

**Issue by dpmontero** [yuku/textcomplete#274]
Hi!
My suggestion is add a new placementStr to placement option: fixed bottom like a dicctionary of phone keyboard. The list rows in one horizontal line.

`var placementStr = 'top|absleft|fixedbottom`

For example:
![keyboard prediction](https://cloud.githubusercontent.com/assets/16160165/16838532/e13a8cb6-49c9-11e6-84f8-0adac41bcd8f.png)

Great job!

**Comment by yuku-t** [comment]
This would indeed be good to have.

**Comment by unhammer** [comment]
was this added?

**Comment by yuku** [comment]
Nope. Since this repository had become a mono repository and this issue was written for one of the deprecated projects, it was closed without looking at its contents.
Seems still useful. Reopen.
